test/quantization/core/test_quantized_module.py

```diff
@@ -834,6 +839,66 @@ def test_conv2d_add(self):
                     pad_mode, dilation, X_scale, X_zero_point, W_scale, W_zero_point,
                     Y_scale, Y_zero_point, use_bias, "add", use_channelwise, X2_scale, X2_zero_point)
 
+    @skipIfNoONEDNN
+    def test_conv2d_add_relu(self):
+        """test API functionality for nn.intrinsic.quantized.ConvAdd2d"""
+        with override_quantized_engine('onednn'):
+            options = itertools.product(
+                ["zeros", "reflect"],  # pad_mode
+                [True, False],  # use_bias
+                [True, False],  # use_channelwise
+            )
+            for pad_mode, use_bias, use_channelwise in options:
+                batch_size = 2
+                in_channels_per_group = 2
+                H = 8
+                W = 8
+                out_channels_per_group = 2
+                groups = 3
+                kernel_h = 3
+                kernel_w = 3
+                stride_h = 2
+                stride_w = 2
+                pad_h = 1
+                pad_w = 1
+                dilation = 1
+                # Tests the correctness of the conv2d module.
+                in_channels = in_channels_per_group * groups
+                out_channels = out_channels_per_group * groups
+                input_feature_map_size = (H, W)
+                kernel_size = (kernel_h, kernel_w)
+                stride = (stride_h, stride_w)
+                padding = (pad_h, pad_w)
+                dilation = (dilation, dilation)
+                X_scale = 1.3
+                X_zero_point = 2
+                X2_scale = 1.2
+                X2_zero_point = 1
+                W_scale = [0.5]
+                W_zero_point = [0] if qengine_is_onednn() else [3]
+                Y_scale = 5.0
+                Y_zero_point = 4
```

Comment:
nit: please move these outside of the loop

Reply:
Thanks and changed.